ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -600,13 +600,24 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+::v-deep .theme--light.v-icon.mdi-close {
+  color: $primary-blue !important;
+}
+
+.solid {
+  border: 0;
+  border-top: 0.25px solid $gray4 !important;
+}
 
 u {
     border-bottom: 1px dotted #000;
     text-decoration: none;
 }
 
 #addHomeOwnerForm {
+  .gray {
+    color: $gray7 !important;
+  }
```

Comment:
Perhaps follow the same approach done with the class `.gray6` in base.scss. 
ie create a global class similar, just might make your life easier in the long run, while also specifying the gray type.

Reply:
Looks like `gray7` already exists locally in some components, so there is definitely a use case for this globally (in base.scss). 
Could you also refactor those local versions of the class as well, see HomeOwnersReview and TransferDetailsReview

Reply:
Good idea, will do

Reply:
Looks like its just HomeOwnersReview and TransferDetailsReview that use it, changed for those as well

Reply:
Right on, thanks!